### PR TITLE
Adding1InEachElementInAnArrayUsingMethodsTask

### PR DESCRIPTION
**Pull request by MeiRR17**

### CODE REVIEW

File: Adding1InEachElementInAnArrayUsingMethodsTask.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+public class Adding1InEachElementInAnArrayUsingMethodsTask {
+    public static void main(String[] args) {
+        //print what you want to say without moving to the next line:
+        System.out.println("Printing all the array but we will add one to each element: ");
+        //Put an arr in main:
+        int [] arrFinal = {1, 2, 3, 4, 5};
```
Comment (anz-davar) [inline Adding1InEachElementInAnArrayUsingMethodsTask.java:6]: name is not correct its not final its your input array

File: Adding1InEachElementInAnArrayUsingMethodsTask.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+public class Adding1InEachElementInAnArrayUsingMethodsTask {
+    public static void main(String[] args) {
+        //print what you want to say without moving to the next line:
+        System.out.println("Printing all the array but we will add one to each element: ");
+        //Put an arr in main:
+        int [] arrFinal = {1, 2, 3, 4, 5};
+        //Save the array from method in main, so we can perform in main the printing:
+        int [] iFinal = addOne(arrFinal);
```
Comment (anz-davar) [inline Adding1InEachElementInAnArrayUsingMethodsTask.java]: you can call that arrFinal iFinal does not make any sense

**Review by anz-davar — CHANGES_REQUESTED** [review]
change names

**Comment by MeiRR17** [comment]
OK


‫בתאריך יום ה׳, 18 באוג׳ 2022 ב-15:51 מאת ‪anz-davar‬‏ <‪
***@***.***‬‏>:‬

**Pull request merged**